tools/depends/download-files.include

```diff
@@ -49,9 +50,12 @@ ifeq ($(HASH_FOUND),yes)
               echo "download $(ARCHIVE) retry $$tries" ;\
               rm $(TARBALLS_LOCATION)/$(ARCHIVE) ;\
               sleep 3 ;\
-              $(RETRIEVE_TOOL) $(RETRIEVE_TOOL_FLAGS) $(BASE_URL)/$(ARCHIVE) ;\
+              $(RETRIEVE_TOOL) $(RETRIEVE_TOOL_FLAGS) -v $(BASE_URL)/$(ARCHIVE) ;\
```

Comment:
put -v in RETRIEVE_TOOL_FLAGS or is this only temporary?

Reply:
My thinking is that we don't really want a verbose output for every curl download, we only really care if we are having to re-download because of a failure, therefore we only add here.
The verbose output is VERY detailed, and really isn't useful if it just works first time

Reply:
yes, only be verbose when it fails